src/Entity/LineItem.php

```diff
@@ -42,6 +42,9 @@ class LineItem implements LineItemInterface, ResourceInterface
     /** @var int */
     protected $subtotal;
 
+    /** @var string|null */
+    protected $taxRate;
```

Comment:
What about change in orm mapping? And there should be probably a migration if it is possible 🤔